src/connections-glue.c

```diff
@@ -10,6 +10,7 @@
 #ifdef USE_OPENSSL
 # include <openssl/ssl.h>
 # include <openssl/err.h>
+# include <openssl/sha.h>
```

Comment:
This was supposed to be added to `src/connections.c` 😅 (although perhaps the corresponding code should be moved into `src/network_i2p.c` now that it exists?)